### DISCUSSION TITLE
Traffic light Chart (for machine state)

### DISCUSSION THREAD

**Discussion by Ihebbenali95** [apache/superset#28414] [Ideas]
Hello, im running Apache superset on windows and i just want to customise a new chart thats not existing.
i created the needed files (i think) and when i run npm run build i keep getting errors .
Does anyone have an idea how to make that work?
![Screenshot 2024-05-09 200816](https://github.com/apache/superset/assets/164204589/0829aa4c-ce55-4c88-b079-ab4dc5f79b87)
![Screenshot 2024-05-09 200846](https://github.com/apache/superset/assets/164204589/1037da6a-ae3c-4d34-87ff-c577109c833c)

**Comment by mistercrunch** [comment]
Windows isn't officially supported but it should be easy to fire up a VM or use a cloud-provided one
